attestations/RegistryHook.json

```diff
@@ -18,7 +18,7 @@
       "upgradeable": false,
       "pausable": false,
       "licensedModule": false,
-      "erc7562StorageCompliant": true,
+      "erc7562StorageCompliant": false,
```

Comment:
registry hook uses 
```
    mapping(address account => address) public registry;
```

Reply:
True,
I was thinking only about validation phase rules about associated storage.

I also fix other contracts in this category.
Those contracts use accessing storage associated with the account.